korgan/claim_exemplar_style.py

```
@@ -0,0 +1,103 @@
+def install_claim_exemplar_style() -> None:
+    """Install drafting-style and Word-body hooks without changing legal research."""
+    global _INSTALLED, _ORIGINAL_BODY_BLOCKS
+    if _INSTALLED:
+        return
+
+    from korgan import claim_docx
+    from korgan import fast_professional_litigation as litigation
+
+    original_draft: Callable[..., Awaitable[Any]] = litigation.FastProfessionalLitigationService.draft_claim
+    if not getattr(original_draft, "_korgan_exemplar_style", False):
+        async def styled_draft(self: Any, case_context: str, research: Any, language: str = "ru") -> Any:
```

Ruff (0.16.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `self`

(ANN401)